To: vim_dev@googlegroups.com
Subject: Patch 7.3.302
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.302 (after 7.3.301)
Problem:    Test 19 fails without 'smartindent' and +eval.
Solution:   Don't use ":exe". Source small.vim.
Files:	    src/testdir/test19.in


*** ../vim-7.3.301/src/testdir/test19.in	2011-09-07 19:58:04.000000000 +0200
--- src/testdir/test19.in	2011-09-08 20:32:33.000000000 +0200
***************
*** 1,6 ****
--- 1,7 ----
  Tests for "r<Tab>" with 'smarttab' and 'expandtab' set/not set.
  
  STARTTEST
+ :so small.vim
  :set smarttab expandtab ts=8 sw=4
  :" make sure that backspace works, no matter what termcap is used
  :set t_kD=x7f t_kb=x08
***************
*** 14,21 ****
  0wR			
  :" Test that copyindent works with expandtab set
  :set expandtab smartindent copyindent ts=8 sw=8 sts=8
! :exe "norm! o{\<cr>x"
! :?^start?,$w! test.out
  :qa!
  ENDTEST
  
--- 15,22 ----
  0wR			
  :" Test that copyindent works with expandtab set
  :set expandtab smartindent copyindent ts=8 sw=8 sts=8
! o{
! x:?^start?,$w! test.out
  :qa!
  ENDTEST
  
*** ../vim-7.3.301/src/version.c	2011-09-07 19:58:04.000000000 +0200
--- src/version.c	2011-09-08 23:22:01.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     302,
  /**/

-- 
Engineers are always delighted to share wisdom, even in areas in which they
have no experience whatsoever.  Their logic provides them with inherent
insight into any field of expertise.  This can be a problem when dealing with
the illogical people who believe that knowledge can only be derived through
experience.
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
